### PR TITLE
(IAC-1169) Prep for 7.2.0 release

### PR DESCRIPTION
**Pull request by michaeltlombardi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,21 @@
...
-## [v7.1.0](https://github.com/puppetlabs/puppetlabs-iis/tree/v7.1.0) (2020-07-07)
+## [v7.2.0](https://github.com/puppetlabs/puppetlabs-iis/tree/v7.2.0) (2020-12-14)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-iis/compare/v7.1.0...v7.2.0)
+
+### Added
+
+- pdksync - \(feat\) - Add support for Puppet 7 [\#302](https://github.com/puppetlabs/puppetlabs-iis/pull/302) ([daianamezdrea](https://github.com/daianamezdrea))
+- MODULES-10884 Allow a UNC path as the physical path of a web site [\#301](https://github.com/puppetlabs/puppetlabs-iis/pull/301) ([palintir](https://github.com/palintir))
+- pdksync - \(IAC-973\) - Update travis/appveyor to run on new default branch `main` [\#286](https://github.com/puppetlabs/puppetlabs-iis/pull/286) ([david22swan](https://github.com/david22swan))
```
Comment (DavidS) [inline CHANGELOG.md]: edited the labels of this PR so it doesn't show up in the Changelog. this is only a maintenance update.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,21 @@
...
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [v7.1.0](https://github.com/puppetlabs/puppetlabs-iis/tree/v7.1.0) (2020-07-07)
+## [v7.2.0](https://github.com/puppetlabs/puppetlabs-iis/tree/v7.2.0) (2020-12-14)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-iis/compare/v7.1.0...v7.2.0)
+
+### Added
+
+- pdksync - \(feat\) - Add support for Puppet 7 [\#302](https://github.com/puppetlabs/puppetlabs-iis/pull/302) ([daianamezdrea](https://github.com/daianamezdrea))
```
Comment (DavidS) [inline CHANGELOG.md]: edited this PR to remove the redundant `feat` and `pdksync` tags in the title.

**Review by DavidS — APPROVED** [review]
please re-run the changelog generator to capture the edited PRs. otherwise looks good to me

**Comment by sanfrancrisko** [comment]
Given `7.2.0` is released, this PR no longer seems valid.

**Pull request closed without merging**